testscript/script_test.go

```diff
@@ -34,6 +34,15 @@ var (
 	binPath string
 )
 
+func PrepareBuildCommand(binPath string) *exec.Cmd {
+	_, disableRaceSet := os.LookupEnv("DISABLE_RACE_CHECKS")
```

Comment:
```suggestion
	_, disableRaceSet := os.LookupEnv("SOFT_SERVE_DISABLE_RACE_CHECKS")
```

Reply:
Done.

It's not being used anywhere, but without it I can't run the tests locally on Windows.